### PR TITLE
Login page branch

### PR DESCRIPTION
**Pull request by yakupcanatalar**
Implemened Login page

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,35 +1,17 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+//import { useState } from 'react'
```
Comment (yakupcanatalar) [inline src/App.tsx]: Remove unused code blocks and imports. 
No needed.

File: src/components/Login.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
+body {
```
Comment (yakupcanatalar) [inline src/components/Login.css]: is this main body? If yes, i think, it should not be here, you should locate in App.css

File: src/App.tsx (hunk 1 above)
Comment (CagriAtalar) [reply src/App.tsx]: Unused code blocks and imports removeo safely.

File: src/components/Login.css (hunk 2 above)
Comment (CagriAtalar) [reply src/components/Login.css]: This is main body for the Login component. Due to our login page is the same as main page, we can do both. I have just added to App.css just now without errors.

**Pull request merged**